lib/KSelect/index.vue

```diff
@@ -70,6 +76,7 @@
           <UiIcon
             v-if="!clearableState"
             class="ui-select-dropdown-button"
+            :style="{ activeColorStyle }"
```

Comment:
`activeColorStyle` is already an object, so the curly brackets around it here can be removed